internal/learn/contribute-to-wiki/how_tos/how_to_write_how_tos.md

```diff
@@ -0,0 +1,90 @@
+---
+keywords:
+- how to
+description: How to write good how tos for the IOTA Wiki.
+image: /img/iota-wiki.png
+---
+# How To Write Banger How Tos
+
+A person would read a how to guide to learn how to solve a real task at their hands, and different people would perform the same task under different circumstances. Even a trivial thing like connection to a node can be thwarted by unstable internet connection, firewall settings, or the node not being healthy.
+
+A how to guide helps readers to achieve their goal. It explains how to use a function of the product in a safe way to get a desired result. A big part of it is explaining what to look at before using the product, and helping to interpret the output to see if everything went as the user have intended.
+
+Keep how tos atomic: let each describe one particular task at a time. It is easier for the reader to compose multiple smaller guides to solve a larger issue than to extract a smaller bit out of a long and complex article. And it's easier to write multiple small articles over a long read.
```

Comment:
```suggestion
### Keep How Tos Atomic

Let each describe one particular task at a time. It is easier for the reader to compose multiple smaller guides to solve a more significant issue than to extract a minor bit from a long and complex article. And it's easier to write multiple small articles.
```

Reply:
I'd rather keep it as a plain paragraph.